src/openrct2-ui/scripting/ScWidget.hpp

```diff
@@ -500,16 +501,30 @@ namespace OpenRCT2::Scripting
             auto widget = GetWidget();
             if (widget != nullptr && widget->type == WindowWidgetType::FlatBtn)
             {
+                if (GetTargetAPIVersion() <= API_VERSION_63_G2_REORDER)
+                {
+                    return LegacyIconIndex(widget->image);
+                }
                 return widget->image;
             }
             return 0;
         }
-        void image_set(uint32_t value)
+        void image_set(DukValue value)
         {
             auto widget = GetWidget();
             if (widget != nullptr && widget->type == WindowWidgetType::FlatBtn)
             {
-                widget->image = value;
+                if (value.type() == DukValue::Type::NUMBER)
+                {
+                    if (GetTargetAPIVersion() <= API_VERSION_63_G2_REORDER)
```

Comment:
Shouldn’t this be

```suggestion
                    if (GetTargetAPIVersion() < API_VERSION_63_G2_REORDER)
```
?